ingest/src/main/java/com/microsoft/azure/kusto/ingest/ManagedStreamingIngestClient.java

```diff
@@ -393,23 +406,31 @@ protected IngestionResult ingestFromStreamImpl(StreamSourceInfo streamSourceInfo
             sourceId = UUID.randomUUID();
         }
 
+        streamSourceInfo.setSourceId(sourceId);
         byte[] streamingBytes;
+        ByteArrayInputStream byteArrayStream;
+
+        if (queuingPolicy.shouldUseQueuedIngestion(streamSourceInfo.getStream().available(), streamSourceInfo.getRawSizeInBytes(),
+                streamSourceInfo.getCompressionType() != null, ingestionProperties.getDataFormat())) {
+            log.info("Stream size ({} bytes) is greater than max streaming size according to policy. Falling back to queued.",
+                    streamSourceInfo.getRawSizeInBytes() > 0 ? streamSourceInfo.getRawSizeInBytes() : streamSourceInfo.getStream().available());
```

Comment:
This log and check are again copy-pasted from elsewhere

Reply:
log yes - not code